### DISCUSSION TITLE
Returning a value from Shortcuts action

### DISCUSSION THREAD

**Discussion by pointlessone** [RubyistApp/Community#5] [Q&A]
I'd love to use Ruby scripts to do complex data transformations in Shortcuts. I figured out that ARGV holds the action input. But I can't figure out how to return a value from a script back to Shortcuts. Any pointers?

**Comment by dragoncoder047** [comment]
I haven't used Rubyist much but I think that you can use `puts` or some other way to print the data to stdout. ![image](https://github.com/RubyistApp/Discussions/assets/101021094/05800789-6b47-4de1-90f2-85df20639f15)
